thirdeye/thirdeye-frontend/app/pods/services/duration/service.js

```diff
@@ -0,0 +1,65 @@
+import Service from '@ember/service';
+import { assert } from '@ember/debug';
+import { isPresent } from "@ember/utils";
+
+export default Service.extend({
+  durationObj: null,
+  durationTypes: {
+    duration: 'string',
+    startDate: 'number',
+    endDate: 'number'
+  },
+
+  init() {
+    this._super(...arguments);
+    this.set('durationObj', {});
```

Comment:
If you doing this. You can remove the declaration `durationObj: null,` i believe.

Reply:
Yes - makes sense. I noticed we're doing it [here in RCA](https://github.com/linkedin/pinot/blob/b40464de8752453ac61ba5a69723977fae8643f0/thirdeye/thirdeye-frontend/app/pods/services/rootcause-aggregates-cache/service.js#L22) as well.